ProjectLighthouse/Database.cs

```diff
@@ -47,6 +47,8 @@ public class Database : DbContext
     public DbSet<EmailVerificationToken> EmailVerificationTokens { get; set; }
     public DbSet<EmailSetToken> EmailSetTokens { get; set; }
 
+    public DbSet<PasswordResetToken> PasswordResetTokens { get; set; }
```

Comment:
This file (in particular the giant blob of `DbSet`s) could use some cleanup and I recognize that, but for now just stick to the regular formatting:
```suggestion
    public DbSet<PasswordResetToken> PasswordResetTokens { get; set; }
```